cicd/pipelines/op-release-notes.adoc

```diff
@@ -23,6 +23,8 @@ include::modules/op-tkn-pipelines-compatibility-support-matrix.adoc[leveloffset=
 include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
 // Modules included, most to least recent
+// include::modules/op-release-notes-1-8.adoc[leveloffset=+1]
```

Comment:
This line is intentionally commented out. @sounix000 will uncomment it when Pipelines 1.8 goes live (after OCP 4.11 GA).